plugins/modules/mso_backup.py

```diff
@@ -45,6 +45,14 @@
     description:
     - Brief information about the backup.
     type: str
+  file_path_name:
```

Comment:
can we call it "file" or use the "backup" attribute for it?

Reply:
ok